src/model/schema.js

```diff
@@ -1083,6 +1122,15 @@ export class SchemaContext {
 		return this._items[ Symbol.iterator ]();
 	}
 
+	/**
+	 * Adds new item at the to of the context.
+	 *
+	 * @param {module:engine/model/node~Node|String} item Item to add.
+	 */
+	addItem( item ) {
```

Comment:
See https://github.com/ckeditor/ckeditor5-engine/issues/1250 and comments about immutable context.

Reply:
Still open issue.

Reply:
eh... probably lost my change after merge.

Reply:
Nope, it's github cache.